src/adblockcss.js

```diff
@@ -0,0 +1,169 @@
+import { AsyncIterReader } from "warcio";
+
+export async function getAdBlockCSSResponse(fullDomain, adblockUrl) {
+  const domainParts = fullDomain.split(".");
+  const allDomains = [];
+
+  for (let i = 0; i < domainParts.length - 1; i++) {
+    if (domainParts[i] !== "www") {
+      allDomains.push(domainParts.slice(i).join("."));
+    }
+  }
+  const domain = allDomains.length ? allDomains[allDomains.length - 1] : "";
```

Comment:
I wondered if this would be an issue with `.co.uk` sites, but seems to be working well with the examples I found in easylist

Reply:
yes, that's a good point, re: TLDs. Not sure if it is significant here, as it does check all subdomains for an exact match, but process the line if it finds the 'domain' match, so it might just process more lines initially.
The best solution would be to also check against a TLD list, but not sure if we need this yet. Will add a comment.